crates/storage-api/src/partition_store/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod protobuf_types;
```

Comment:
Wondering why do we need the "partition_store" module. Can we instead include the `protobuf_types.rs` directly under the `storage-api` so it's `storage_api::protobuf_types` instead of `storage_api::partition_store::protobuf_types`?